Makefile

```diff
@@ -44,33 +44,33 @@ clean:
 test: test-unit test-integration
 
 test-unit:
-	go test -race -short -v ./...
+	go test -failfast -race -short -v ./...
 
 test-unit-postgres:
-	go test -race -tags postgres -short -v ./...
+	go test -failfast -race -tags postgres -short -v ./...
 
 test-unit-cover:
-	go test -short -v -coverprofile=coverage.txt -covermode=count -coverpkg=all $(go list ./... | grep -v /demo/)
+	go test -failfast -short -v -coverprofile=coverage.txt -covermode=count -coverpkg=all $(go list ./... | grep -v /demo/)
 
 test-unit-postgres-cover:
-	go test -short -tags integration -v -coverprofile=coverage-postgres.txt -covermode=count -coverpkg=all $(go list ./... | grep -v /demo/)
+	go test -failfast -short -tags integration -v -coverprofile=coverage-postgres.txt -covermode=count -coverpkg=all $(go list ./... | grep -v /demo/)
 
 test-integration:
-	go test -v ./demo
+	go test -failfast -v ./demo
 	cd demo && go build && ./demo -build -test -debug
 
 test-integration-postgres:
-	go test -v ./demo
-	go test -race -short -tags integration -v ./...
+	go test -failfast -v ./demo
+	ge test -failfast -race -short -tags integration -v ./...
```

Comment:
```suggestion
	go test -failfast -race -short -tags integration -v ./...
```

Reply:
Yup saw that too but Github messed up my branch somehow and now I also have a Doc update and will need to rebase and force push. Weird